test-cargo-miri/run-test.py

```diff
@@ -180,6 +185,11 @@ def test_cargo_miri_test():
         default_ref, "test.stderr-empty.ref",
         env={'MIRIFLAGS': "-Zmiri-permissive-provenance"},
     )
+    test("`cargo miri test` (multiple targets)",
+        cargo_miri("test", targets = ["aarch64-unknown-linux-gnu", "s390x-unknown-linux-gnu"]),
+        "test.multiple_targets.stdout.ref", "test.stderr-empty.ref",
+        env={'MIRIFLAGS': "-Zmiri-permissive-provenance"},
+    )
```

Comment:
This will make `./run-test.py` take quite a bit longer since it needs to build 2 more sysroots... and also on CI we run this many times with different `--target` but this test here is going to always be the same.

Maybe we should have another flag, like `--multi-target`, and only run the test when the flag is set? Then we can set it on CI for the first `run-test.py` invocation and avoid repeating it later.

Reply:
I added a `--multi-target` flag and made it so that the CI only uses it when testing the host target.

My shell skills are very bad so hopefully the code looks ok, but let me know.